manage.py

```diff
@@ -3,7 +3,9 @@
 import sys
 
 if __name__ == "__main__":
+
```

Comment:
Any specific reason to include these 2 blank lines?

Reply:
No, I think was a typo error!